docs/general/builtins/scalar-functions.rst

```diff
@@ -2718,6 +2718,58 @@ function supports arrays of any of the :ref:`primitive types
     SELECT 1 row in set (... sec)
 
 
+.. _scalar-array_position:
+
+``array_position(anycompatiblearray, anycompatible [, integer ] ) → integer``
+-----------------------------------------------------------------------------
+
+The ``array_position`` function returns the subscript of the first
+occurrence of the second argument in the ``array``, or ``NULL`` if it's not
+present. If the third argument is given, the search begins at that subscript.
+The ``array`` must be one-dimensional. Comparisons are done using
+``IS NOT DISTINCT FROM`` semantics, so it is possible to search for ``NULL``.
+
+::
+
+    cr> CREATE TABLE metrics(
+            ts timestamp,
+            values_sensor1 ARRAY(INTEGER),
+            values_sensor2 ARRAY(INTEGER)
+        );
```

Comment:
Looks like the `ts` column is not needed for the examples, so I suggest to remove (keep scope low). Wouldn't also one array column be enough for all examples?

Reply:
I have simplified the function documentation. Removed verbose details taking inspiration from other functions. Please take a look again.